code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -119,13 +119,18 @@ assumpF otherContents = SRS.assumpt (assumpIntro : otherContents) []
 
 -- | Wrapper for 'thModelIntro'. Takes the program name and other 'Contents'.
 thModF :: (Idea a) => a -> [Contents] -> Section
-thModF progName otherContents = SRS.thModel (thModIntro progName : otherContents) []
+thModF _        []                   = SRS.thModel [thModIntroNoContent] []
+thModF progName otherContents = SRS.thModel (thModIntro progName : 
+                                              otherContents) []
 
 -- | Creates a eneralized Theoretical Model introduction given the program name.
 thModIntro :: (Idea a) => a -> Contents
 thModIntro progName = foldlSP [S "This", phrase section_, S "focuses on the",
   phrase general, plural equation `S.and_` S "laws that", short progName, S "is based on"]
 
+thModIntroNoContent :: Contents
+thModIntroNoContent = mkParagraph $ S "There are no theoretical models."
```

Comment:
This is a 'pattern', i.e "There are no" followed by the name of the part you're looking at. There should be chunks for each (theoretical model, etc) from which you can pull the name.

We don't want to introduce multiple strings "theoretical model" in our code base.

Reply:
Ok, I will change it!